src/scenes/ProgressReports/StatusHistory/StatusHistory.tsx

```diff
@@ -0,0 +1,117 @@
+import { useQuery } from '@apollo/client';
+import {
+  Box,
+  Breadcrumbs,
+  FormControlLabel,
+  Skeleton,
+  Switch,
+  Typography,
+} from '@mui/material';
+import { useToggle } from 'ahooks';
+import { Helmet } from 'react-helmet-async';
+import { flexColumnStyle } from '~/common';
+import { Breadcrumb } from '~/components/Breadcrumb';
+import { idForUrl, useChangesetAwareIdFromUrl } from '~/components/Changeset';
+import { EngagementBreadcrumb } from '~/components/EngagementBreadcrumb';
+import { ReportLabel } from '~/components/PeriodicReports/ReportLabel';
+import { ProjectBreadcrumb } from '~/components/ProjectBreadcrumb';
+import { Navigate } from '~/components/Routing';
+import { Error } from '../../../components/Error';
+import {
+  ProgressReportDetailDocument,
+  ProgressReportDetailFragment,
+} from '../Detail/ProgressReportDetail.graphql';
+import { WorkFlowEventHistory } from '../Detail/WorkflowEventsHistory';
+
+export const StatusHistory = () => {
+  const { id, changesetId } = useChangesetAwareIdFromUrl('reportId');
+  const [checked, set] = useToggle();
+
+  const { data, error } = useQuery(ProgressReportDetailDocument, {
+    variables: {
+      id,
+      changesetId,
+    },
+  });
+  if (error) {
+    return (
+      <Error page error={error}>
+        {{
+          NotFound: 'Could not find status history',
+          Default: 'Error loading status history',
```

Comment:
It's the progress report that could not be found. I would change the error message to that.
```suggestion
          NotFound: 'Could not find quarterly report',
          Default: 'Error loading quarterly report's status history',
```